doc/user_guide/parameters.tex

```diff
@@ -736,6 +740,31 @@ \section{Job Control}
 
 The default value of this parameter is $\verb#false#$.
 
+\subsection[scdm\_proj]{\tt logical :: scdm\_proj}
+If {\tt scdm\_proj=true} then the $A_{mn}^{(\mathbf{k})}$ matrices are generated with the SCDM-k method of Ref.~\cite{LinLin-ArXiv2017}. In this case, one also needs to specify the other three {\tt scdm} keywords. One then needs to run {\tt wannier90.x -pp seedname} to generate the {\tt seedname.nnkp} file, to be used by a first-principle code (at the moment only interface available is with the QuantumEspresso code). It is not possible to set {\tt scdm\_proj=true} and simultaneously specify a projection block.
```

Comment:
Are mu and sigma really always needed to be specified, also in the isolated case?

Reply:
No they don't you are right. I have modified the text according to the logic.